debian/copyright

```diff
@@ -0,0 +1,61 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: rocksdb
+Upstream-Contact: G. B. Versiani <guibv@yahoo.com>
+Source: http://rocksdb.org
+
+Files: debian/*
+Copyright:
+  2015 G. B. Versiani <guibv@yahoo.com>
+License: GPL-3
```

Comment:
Does this really need to be GPL-3? I think it would be better not to introduce another licence (beyond BSD)